frontend/src/components/chart/MultiYAxesChart.vue

```diff
@@ -140,19 +146,56 @@ export default {
           },
         },
         labels: timeLabels,
-        yaxis: {
-          title: {
-            text: "Average Power (kW)",
+        yaxis: [
+          {
+            title: {
+              text: "Average Power (kW)",
+            },
+            seriesName: "Battery Charging (kW)",
           },
-          min: 0,
-        },
+          {
+            seriesName: "Battery Charging (kW)",
+            show: false
```

Comment:
is there a reason for having 7 equal dictionary elements {seriesName: "Battery Charging (kW)",
            show: false}?

Reply:
Per the documentation (https://apexcharts.com/docs/chart-types/multiple-yaxis-scales/), each index of Y-axis corresponds to the index of series. In order to have the final emission series corresponding with the emission Y-axis, we need to add the equal dictionary elements in front to make their indexes equal. The example code in the documentation is also doing something similar to that.